### ISSUE TITLE
Can't access to the level.dat file in Minecraft 1.16.1 

### ISSUE THREAD

**Issue by Phantam1num** [jaquadro/NBTExplorer#100]

**Comment by rebane2001** [comment]
Please provide more information

**Comment by LuvbCa** [comment]
and 1.15.2 files work?

**Comment by CptReddot** [comment]
I have the same problem. Can't open level.dat, but all other xxx.dat- files. And yes, 1.15. works fine. I think it's time for an update...

**Comment by DrDoom347** [comment]
Running into this issue as well. Anyone have a current workaround to edit the level.dat file? I need to access it in order to override the player data... because Mojang's player data system is still broken (SP data always takes precedence when playing a previous MP save file in SP)

**Comment by JuniorWolfgamingDE** [comment]
I even have the problem. Older versions works, 1.16 does not.

**Comment by CDWimmer** [comment]
This is still the case, judging by @jaquadro 's contribs they are very busy, but still I'd like to ask if there's any plan to update this for 1.16.x compatability given its coming up on 4 years since the last update and I've failed to find a competing tool

**Comment by 0xfeeddeadbeef** [comment]
I use **[nbted](https://github.com/C4K3/nbted)** as a workaround; it's a command line utility (No fancy GUI) for exporting NBT files to textual representation; edit the text file using your favorite editor and then convert it back to binary NBT using the same utility.

```bash
nbted --print level.dat > level.txt
vim level.txt
nbted --reverse level.txt > level.dat
```

**Comment by CDWimmer** [comment]
Thanks, I'm trying to copy specific chunks from one world to another which NTBEx was capable of, I'm not sure how I'd go about that with just the command line!

**Comment by 0xfeeddeadbeef** [comment]
@CDWimmer You're right, nbted will not be useful in your case. Try [MCA Selector](https://github.com/Querz/mcaselector) instead.

**Comment by DrDoom347** [comment]
> Thanks, I'm trying to copy specific chunks from one world to another which NTBEx was capable of, I'm not sure how I'd go about that with just the command line!

A tool that may be capable of that is MCA Selector. I've used it to manipulate chunks and it does work on current versions (I think even the 1.17 snapshots)

Oops I was a moment too late lmao yeah what 0x said

**Comment by FoxieFlakey** [comment]
1.16.5 level.dat file works perfectly with NBTExplorer

**Comment by Meeples10** [comment]
I can't reproduce this issue. NBTExplorer 2.8.0 opens level.dat files from 1.16 and 1.17 worlds without any problems.
